### DISCUSSION TITLE
硕士论文要求：表格 与正文之间 需要空一行，

### DISCUSSION THREAD

**Discussion by Ethan1239** [note286/xduts#191] [Q&A] [answered]
```latex
图\ref{信号处理板实物图}中的各个部件以及具体功能描述见表\ref{信号处理板功能描述表}。

\begin{table}[!ht]
\caption{信号处理板功能描述表}
\label{信号处理板功能描述表}
\begin{tblr}
{
hlines,vlines,
columns = {valign=m,co=-1},
rows    = {halign=c},
width = 0.8\linewidth, % 控制表格宽度为行宽,
}
编号 & 主要器件 & 功能描述 \\
1    & 异构 RFSoC & 切片式干扰模拟、杂波模拟\\
2    & 高性能 FPGA & 压制式干扰模拟\\
3    & 射频通道 & 射频信号发射与采集 \\
4    & 背板接口 & 与子板接口互联\\
5    & DDR & 大容量存储芯片\\
\end{tblr}
\end{table}
```

**Comment by Ethan1239** [comment]
![2024-05-21_22-11-52](https://github.com/note286/xduts/assets/94029460/c59ab7cb-4960-481c-9dda-1f2e53bbfdad)

**Comment by note286** [comment]
你可以看一下Microsoft Word的示例，并没有真的所谓的空一行；其次，空一行的这个概念是很模糊的，是空字高还是算行间距？

另外，目前的样式是可以毕业的，去年也这样，你倒是不用担心。

**Reply by Ethan1239** [reply]
答辩组专家给提的意见，我这还要必须修改。

**Reply by note286** [reply]
但问题是这与学位办的要求相悖啊，你去看无论是浮动体上方还是下方与正文的间距，与Microsoft Word给出的示例均一致。

**Reply by Ethan1239** [reply]
我看学位办给的是这个样式
![表格样式](https://github.com/note286/xduts/assets/94029460/8723b202-9293-4b1b-a731-d6be2833cc9d)

**Reply by note286** [reply, marked as answer]
左侧的是xduts的效果，右侧是Microsoft Word里的，间距是一模一样的，两边的红色矩形是复制粘贴一模一样的。

![image](https://github.com/note286/xduts/assets/16207292/f6e74a2d-9395-4ae0-80b8-d2516d812f82)

**Reply by Ethan1239** [reply]
是的，但是他前语句说出"表格与正文之间一般应空一行”。

**Reply by Ethan1239** [reply]
最后使用魔法打败魔法。使用了“\vspace{}语句”

**Comment by sikouhjw** [comment]
参考 https://github.com/sikouhjw/gdutthesis/blob/0f3745127922c301296a210dcdcd5de119c00dce/gdutthesis.cls#L1148-L1174
